src/pages/Jiyoung/pages/Login/Login.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Button from '../../components/Button/Button';
+import '../../scss/style.scss';
+
+const Login = () => {
+  const navigate = useNavigate();
+  const [userId, setUserId] = useState('');
+  const [userPw, setUserPw] = useState('');
+  const emailPattern =
+    /^([0-9a-zA-Z_.-]+)@([0-9a-zA-Z_-]+)(\.[0-9a-zA-Z_-]+){1,2}$/;
+
+  const joinBtn = () => {
+    navigate('/jiyoung-join');
+  };
+
+  const saveUserId = (event) => {
+    setUserId(event.target.value);
+  };
+
+  const saveUserPw = (event) => {
+    setUserPw(event.target.value);
+  };
+
+  const emailCheck = () => {
+    if (emailPattern.test(userId)) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  const isUserInputValid = emailCheck() && userPw.length >= 5;
+
+  return (
+    <div className="page">
+      <div className="content login">
+        <div className="contentBody">
+          <div className="loginLogo">
+            <span className="blind">wecode</span>
+          </div>
+        </div>
+        <div className="contentFooter">
+          <div className="loginForm">
+            <form>
+              <div className="formInput">
+                <label htmlFor="inputEmail" className="formLabel">
+                  <span className="blind">이메일</span>
+                </label>
+                <input
+                  id="inputEmail"
+                  type="email"
+                  className="formControl"
+                  placeholder="이메일"
+                  onChange={saveUserId}
+                  value={userId}
+                />
```

Comment:
혹은 함수에 인자를 전달할 때 함수내부에서 event.target에 대한 value값만 사용한다면
handle함수를 호출할 때 event.target.value만을 넘겨줄 수도 있습니다
```suggestion
                <input
                  id="inputEmail"
                  type="email"
                  className="formControl"
                  placeholder="이메일"
                  onChange={(event)=>saveUserId(event.target.value)}
                  value={userId}
                />
```